src/docfx/Errors.cs

```diff
@@ -479,6 +479,27 @@ public static Error MonikeRangeOutOfScope(SourceInfo<string?> configMonikerRange
                 => new Error(ErrorLevel.Error, "moniker-range-out-of-scope", $"No moniker intersection between docfx.yml/docfx.json and file metadata. Config moniker range '{configMonikerRange}' is {StringUtility.Join(configMonikers)}, while file monikers is {StringUtility.Join(fileMonikers)}.", source);
         }
 
+        public static class ZonePivot
+        {
+            public static Error ZonePivotGroupDefinitionNotFound(FilePath file)
```

Comment:
This error does not appear in the feature spec, is it the same as `pivot-group-not-found`?

Reply:
Yes, this is not in the feature spec, and it's for the case where `zone_pivot_groups` presents but we can't find `zone-pivot-groups.json` or `zone_pivot_groups_filename` in metadata.

Reply:
For that we have a general `file-not-found` error.